site/static/js/plugins.js

```diff
@@ -43,6 +43,23 @@ function convertFromHugo(content) {
   return content.replace(hugoHideDirectives, "").replace(hugoDirectiveTags, "");
 }
 
+async function addVersionDropdown() {
+  const sidebar = document.getElementsByClassName("sidebar").item(0);
+  const versionDropdown = `
+  <div class="dropdown">
+                <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">v1.0.0-alpha.3
+                <span class="caret"></span></button>
+                <ol class="dropdown-menu">
+                <li><a href="/installation/">v1.0.0-alpha.3</a></li>
```

Comment:
this is OK for now but I think the full implementation of a version drop down would preserve the current page and maybe just modify the URL or something along those lines.

Reply:
I would also maybe name it v1 beta1 or something more durable.

Reply:
Will do for now
We should decide on the branch/tag naming scheme for multiple versions of the documentation and tool we want to show publicly. That'll drive the dropdown entries in the future

Reply:
I missed this. What's `v1beta1`? Please use the release semver.